_posts/2020-11-19-Setting-Up-a-Pull-Request.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Setting Up a Pull Request
+description: 
+summary: 
+tags: 
+---
+Kajari instructed Celeste and I to start creating a new post using pull request. Pull requests are changes committed in the branch that are prresented to viewers of the users repository on Github. To initiate a new post/pull request, the following commands must be used to complete the task.
```

Comment:
Don't start the posts with "Kajari instructed ...". If you want an intro you can say "Today we started working with Pull Requests. Pull requests are changes.."